apps/api/internal/mcp/oauth_test.go

```
@@ -0,0 +1,753 @@
+			q := url.Values{}
+			q.Set("response_type", "code")
+			q.Set("client_id", "client-abc")
+			q.Set("redirect_uri", redirect)
+			q.Set("code_challenge", "abc123challenge")
+			q.Set("code_challenge_method", "S256")
+			if tc.scope != "" {
+				q.Set("scope", tc.scope)
+			}
+
+			w := httptest.NewRecorder()
+			req := httptest.NewRequest(http.MethodGet, "/api/v1/oauth/mcp/authorize?"+q.Encode(), nil)
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)